test/utils/string.formatter.spec.unit.ts

```diff
@@ -31,10 +31,10 @@ describe("String formatter tests", () => {
   });
 
   it("should not remove printable chars", () => {
```

Comment:
I'd suggest adding a unit test where the preFormattedString is a string of the entire set of acceptable characters, and then check the length of the returned string === length of the preFormattedString.

Reply:
i'd agree, we should test all the allowable characters are actually allowed

Reply:
Added the new test.